qualtran/cirq_interop/jupyter_tools.py

```diff
@@ -0,0 +1,115 @@
+#  Copyright 2023 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+from typing import Iterable
+
+import cirq
+import cirq.contrib.svg.svg as ccsvg
+import IPython.display
+import ipywidgets
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+
+import qualtran.cirq_interop.testing as cq_testing
+from qualtran import Register
+from qualtran._infra.gate_with_registers import get_named_qubits, merge_qubits
+from qualtran.cirq_interop import t_complexity_protocol
+
+
+def display_gate_and_compilation(g: cq_testing.GateHelper, vertical=False, include_costs=True):
```

Comment:
Is the plan to keep these around or deprecate in the future?

Reply:
Probably keep them around for now, but happy to deprecate as bloq visualizations mature